src/components/NavRoutes.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Login from "./Login";
+import Dashboard from "./Dashboard";
+import Home from "./Home";
```

Comment:
Bhai I can't see any Home component file

Reply:
See it's here.

![Screenshot 2023-12-12 at 12 13 46 PM](https://github.com/mkh-wtag/blog-app/assets/24565305/5d0c8348-4b8c-45c4-9ee5-276c8b120dec)